pom.xml

```diff
@@ -27,6 +27,9 @@
     <version.spring-cloud-appconfiguration-starter>2.5.0</version.spring-cloud-appconfiguration-starter>
     <version.spring-cloud-feature-management>2.4.0</version.spring-cloud-feature-management>
     <version.azure-spring-boot-starter-keyvault-certificates>3.14.0</version.azure-spring-boot-starter-keyvault-certificates>
+    <version.spring-cloud-azure-native-configuration>4.0.0-beta.1</version.spring-cloud-azure-native-configuration>
+    <version.spring-native>0.11.4</version.spring-native>
+    <repackage.classifier>exec</repackage.classifier>
```

Comment:
why can't we put these properties to the native profile too?

Reply:
I put the spring-native and spring-cloud-azure-native-configuration dependencies in the parent dependency-management. the `repackage.classifier` is used for spring boot maven plugin, it doesn't need to move.

Reply:
Maybe use the spring-native and spring-cloud-azure-native-configuration with the version in each profile dependencies enough.